apps/settings/src/main.js

```diff
@@ -51,5 +53,10 @@ const router = createRouter({
 });
 
 const app = createApp(App);
+
+const pinia = createPinia();
+app.use(pinia);
+useSessionStore(pinia);
```

Comment:
you do not need to initialize the store it. only when you use it :)

you can shorten line 58 by doing app.use(createPinia());

Reply:
Sadly this is needed, it's needs to be initialized before the component is created, otherwise it gives a console error: getActivePinia was called with no active Pinia. Did you forget to install pinia?